src/drivers/sdl/lv_sdl_window.c

```diff
@@ -430,6 +436,31 @@ static void texture_resize(lv_display_t * disp)
 }
 #endif
 
+static void * sdl_draw_buf_realloc_aligned(void * ptr, size_t new_size)
```

Comment:
Should we use lv_draw_buffer_create to take care of alignment automatically?

Reply:
I think I will consider using the draw buf interface next, because it will require processing rotated_buf, which will be a big change. Let's fix the current problem first.